adam-core/src/test/scala/org/bdgenomics/adam/rdd/ADAMContextSuite.scala

```diff
@@ -204,29 +208,29 @@ class ADAMContextSuite extends ADAMFunSuite {
   }
 
   sparkTest("Can read a .gtf file") {
-    val path = testFile("features/Homo_sapiens.GRCh37.75.trun20.gtf")
-    val features: RDD[Feature] = sc.loadFeatures(path)
-    assert(features.count === 15)
+    val path = testFile("Homo_sapiens.GRCh37.75.trun20.gtf")
+    val features: RDD[Feature] = sc.loadFeatures(path).rdd
+    assert(features.rdd.count === 15)
```

Comment:
if `features` is already of type `RDD[Feature]` then why `features.rdd.count` instead of `features.count`?